impeller/typographer/backends/skia/text_frame_skia.cc

```diff
@@ -46,51 +51,61 @@ static Rect ToRect(const SkRect& rect) {
   return Rect::MakeLTRB(rect.fLeft, rect.fTop, rect.fRight, rect.fBottom);
 }
 
-static constexpr Scalar kScaleSize = 64.0f;
-
 std::shared_ptr<TextFrame> MakeTextFrameFromTextBlobSkia(
     const sk_sp<SkTextBlob>& blob,
-    flutter::DlColor dl_color) {
+    flutter::DlColor dl_color,
+    Scalar dx) {
   bool has_color = false;
   Color color = ToColor(dl_color);
   std::vector<TextRun> runs;
   for (SkTextBlobRunIterator run(blob.get()); !run.done(); run.next()) {
     // TODO(jonahwilliams): ask Skia for a public API to look this up.
     // https://github.com/flutter/flutter/issues/112005
+
     SkStrikeSpec strikeSpec = SkStrikeSpec::MakeWithNoDevice(run.font());
     SkBulkGlyphMetricsAndPaths paths{strikeSpec};
 
     const auto glyph_count = run.glyphCount();
     const auto* glyphs = run.glyphs();
     switch (run.positioning()) {
       case SkTextBlobRunIterator::kFull_Positioning: {
-        std::vector<SkRect> glyph_bounds;
-        glyph_bounds.resize(glyph_count);
-        SkFont font = run.font();
-        auto font_size = font.getSize();
-        // For some platforms (including Android), `SkFont::getBounds()` snaps
-        // the computed bounds to integers. And so we scale up the font size
-        // prior to fetching the bounds to ensure that the returned bounds are
-        // always precise enough. Scaling too large will cause Skia to use
-        // path rendering and potentially inaccurate glyph sizes.
-        font.setSize(kScaleSize);
-        font.getBounds(glyphs, glyph_count, glyph_bounds.data(), nullptr);
-
         std::vector<TextRun::GlyphPosition> positions;
         positions.reserve(glyph_count);
         for (auto i = 0u; i < glyph_count; i++) {
           // kFull_Positioning has two scalars per glyph.
           const SkPoint* glyph_points = run.points();
-          const auto* point = glyph_points + i;
+          const SkPoint* point = glyph_points + i;
           Glyph::Type type = paths.glyph(glyphs[i])->isColor()
                                  ? Glyph::Type::kBitmap
                                  : Glyph::Type::kPath;
           has_color |= type == Glyph::Type::kBitmap;
-
+          Point position = Point{point->x(), point->y()} +
+                           Point(run.offset().x(), run.offset().y());
+          position.x += dx;
+          position.x *= 2.625;
+          position.x += 0.125;
+
+          FML_LOG(ERROR) << "Mapped-ish Position: " << position;
+          SubpixelPosition subpixel = SubpixelPosition::kZero;
+          SkIPoint mask = SkIPoint::Make(3, 0);
```

Comment:
Kind of hacked this in there. Need to compute the subpixel position later and remove it from glyph. We can't accurately determine the position during recording.